solr/solr-ref-guide/src/authentication-and-authorization-plugins.adoc

```diff
@@ -81,7 +84,7 @@ This example also defines `security.json` on the command line, but you can also
 
 [WARNING]
 ====
-Depending on the authentication and authorization plugin that you use, you may have user information stored in `security.json`. If so, we highly recommend that you implement access control in your ZooKeeper nodes. Information about how to enable this is available in the section <<zookeeper-access-control.adoc#zookeeper-access-control,ZooKeeper Access Control>>.
+Whenever you use any security plugins and store `security.json` in ZooKeeper, we highly recommend that you implement access control in your ZooKeeper nodes. Information about how to enable this is available in the section <<zookeeper-access-control.adoc#zookeeper-access-control,ZooKeeper Access Control>>.
```

Comment:
I felt that it is important to secure ZK even if you don't have sensitive data in security.json. For the simple reason that if Zk is wide open then *anyone* could disable or change auth*

Reply:
True, good point. +1